adiachenko/src/components/Category/index.tsx

```diff
@@ -33,21 +35,57 @@ const CategoryUl = styled.ul`
   }
 `;
 
-const Category = () => (
-  <CategoryUl>
-    <li className="active">
-      ALL
-    </li>
-    <li>
-      DOCUMENTARY
-    </li>
-    <li>
-      COMEDY
-    </li>
-    <li>
-      CRIME
-    </li>
-  </CategoryUl>
-);
+const categories = [
+    {
+        type: 'All'
+    },
+    {
+        type: 'Documentary'
+    },
+    {
+        type: 'Comedy'
+    },
+    {
+        type: 'Crime'
+    }
+];
+
+interface CategoryLiProps {
+    type: string,
+    category: string,
+    setCategory: () => void
+}
+
+const CategoryLi = ({type, category,  setCategory}: CategoryLiProps) => {
+    const isActive = useMemo(() => type.toLowerCase() === category.toLowerCase(), [type, category]);
+
+    return (
+        <li
+            className={classNames({
+                active: isActive
+            })}
+            onClick={() => setCategory()}
+        >
+            {type}
+        </li>
+)};
+
+const Category = () => {
+    const contextValue = useAppContext();
+
+    return (
+        <CategoryUl>
+            {
+                categories.map(category =>
```

Comment:
Considering `category` object has a single property (`type`), you could've saved some `category.` repetitions with:
```js
categories.map(({category}) =>(
   <CategoryLi
      key={type}
      ..
))
```